package.json

```diff
@@ -1,107 +1,9 @@
 {
-  "name": "tracked-built-ins",
-  "version": "2.0.1",
-  "description": "Tracked versions of JavaScript's built-in classes!",
-  "keywords": [
-    "ember-addon"
+  "private": true,
+  "workspaces": [
+    "addon",
+    "test-app"
   ],
-  "repository": "https://github.com/tracked-tools/tracked-built-ins",
-  "license": "MIT",
-  "author": "Chris Hewell Garrett",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
-  "scripts": {
```

Comment:
@linainu let's keep `scripts` like so:
```json
"scripts": {
    "test": "cd test-app && yarn test"
  },
```

we'll add more once migrate to v2 addon